content/docs/user-guide/gems/reference/aws/aws-gamelift/local-testing.md

```diff
@@ -57,12 +57,9 @@ $ bin\profile\YourProject.GameLauncher.exe --cl_gameliftLocalEndpoint "[http://l
 
```

Comment:
line 52 is wrong, please update to:
$ bin\profile\YourProject.GameLauncher.exe --cl_gameliftLocalEndpoint "http://localhost:9080"